test/unit/argument_parser/detail/format_html_test.cpp

```diff
@@ -29,25 +29,23 @@ TEST(html_test, html)
     argument_parser parser0("empty_options", 3, argv0);
     testing::internal::CaptureStdout();
     EXPECT_EXIT(parser0.parse(), ::testing::ExitedWithCode(EXIT_SUCCESS), "");
-    stdout = testing::internal::GetCapturedStdout();
-    expected = std::string("<!DOCTYPE html PUBLIC \"-//W3C//DTD HTML 4.01//EN\" http://www.w3.org/TR/html4/strict.dtd\">"
-                           "<html lang=\"en\">"
-                           "<head>"
-                           "<meta http-equiv=\"content-type\" content=\"text/html; charset=utf-8\">"
-                           "<title>empty_options &mdash; </title>"
-                           "</head>"
-                           "<body>"
-                           "<h1>empty_options</h1>"
-                           "<div></div>"
-                           "<h2>Synopsis</h2>"
-                           "<h2>Version</h2>"
-                           "<strong>Last update:</strong> <br>"
-                           "<strong>empty_options version:</strong> <br>"
-                           "<strong>SeqAn version:</strong> 3.0.0<br>"
-                           "<br>"
+    std_out = testing::internal::GetCapturedStdout();
+    expected = std::string("<!DOCTYPE html PUBLIC \"-//W3C//DTD HTML 4.01//EN\" http://www.w3.org/TR/html4/strict.dtd\">\n"
```

Comment:
Would be nicer if we could use rapidxml somehow to test the DOM tree. It is not very important but perhaps you can open a card for this?

Reply:
I added a card